src/components/Editor/BottomEditorPanel/RenderError.tsx

```diff
@@ -50,7 +50,7 @@ const RenderError = (props: any) => {
     <Flex sx={styles.root}>
       <Flex sx={styles.list}>
         {list.length > 0
-          ? list.map((item: CadenceProblem, i: number) => {
+          ? list.reverse().map((item: CadenceProblem, i: number) => {
```

Comment:
to avoid array mutation
```suggestion
          ? [...list].reverse().map((item: CadenceProblem, i: number) => {
```
or
```suggestion
          ? list.slice().reverse().map((item: CadenceProblem, i: number) => {
```